OPAL/br/src/main/scala/org/opalj/br/Method.scala

```diff
@@ -468,6 +468,16 @@ final class Method private[br] (
 
     override def asMethod: this.type = this
 
+    /**
+     *
+     * @return wether this class is defined as strict. Starting from Java 17, this is true by default.
```

Comment:
Why adding this here instead of just doing it directly in JVMMethod?

Reply:
JVMMethod does not have access to a classfile